### PR TITLE
Minimap in bottom corner

### PR DESCRIPTION
**Pull request by alexottoboni**
Makes a minimap in the bottom left corner, it is easier to see on bigger screens.

### CODE REVIEW

File: src/game_renderer/GameRenderer.cpp [hunk 1]
```diff
@@ -185,15 +185,244 @@ void GameRenderer::RenderLevelEditor(GLFWwindow* window,
   RenderIt(window, game_state, true);
 }
 
-void GameRenderer::RenderIt(GLFWwindow* window,
+void GameRenderer::RenderMiniMap(GLFWwindow* window,
+                std::shared_ptr<GameState> game_state,
+                bool render_level_editor) {
```
Comment (josepharhar) [inline src/game_renderer/GameRenderer.cpp:190]: almost all of this method is a copy of the main render method, right? could we consolidate it into one method?

Comment (alexottoboni) [reply src/game_renderer/GameRenderer.cpp:190]: A way I could see consolidating it is pulling each type of item draw routine out into its own method. But that would be a lot of tiny functions.

Comment (bbeck13) [reply src/game_renderer/GameRenderer.cpp:190]: I'd like that but I think that's out of scope for this commit https://github.com/bbeck13/RhythmRunner/issues/134

**Comment by bbeck13** [comment]
CR 👃 QA 👣

**Pull request merged**